manifests/proxydhcp.pp

```diff
@@ -49,6 +49,25 @@
     failover       => $failover,
   }
 
+
+  if $foreman_proxy::dhcp_manage_acls and $::osfamily == 'RedHat' {
+
+    package {'acl':
+      ensure => 'installed',
+    }
+    -> exec { 'setfacl_etc_dhcp':
+      command => "setfacl -R -m u:${::foreman_proxy::user}:rx /etc/dhcp",
+      path    => '/usr/bin',
+      onlyif  => "getfacl -p /etc/dhcp | grep \"owner: ${::foreman_proxy::user}\" && getfacl -p /etc/dhcp | grep \"user::r-x\"",
```

Comment:
Is that second grep needed? I thought that was for the actual owner and we can assume that's correct I think.

Reply:
I dunno, I assumed that `u:foreman-proxy:rx` was setting permissions on the directory and that'd want to ensure they got set properly? I am a ACL nub.

Reply:
```user::r-x``` will be there if you do ```chmod u=rx /etc/dhcp```. In ACLs ```user::```, ```group::``` and ```other::``` are the classic permissions we all know and use. The ```user:foreman-proxy:r-x``` are additional permissions for the user foreman-proxy.

In short: ```unless => "getfacl -p /etc/dhcp | grep user:${::foreman_proxy::user}:r-x"``` should do the trick.